passthroughserver/main.cpp

```diff
@@ -23,64 +23,33 @@
 #include "waylandplatform.h"
 #endif // HAVE_WAYLAND
 #include "unknownplatform.h"
+#include "logging.h"
 
 #include <QGuiApplication>
 #include <QtDebug>
 
 namespace {
+QLoggingCategory::CategoryFilter defaultLoggingFilter;
 
-void disableMInputContextPlugin()
-{
-    // none is a special value for QT_IM_MODULE, which disables loading of any
-    // input method module in Qt 5.
-    setenv("QT_IM_MODULE", "none", true);
-}
-
-bool isDebugEnabled()
+void loggingCategoryFilter(QLoggingCategory *category)
 {
-    static int debugEnabled = -1;
-
-    if (debugEnabled == -1) {
+    if (qstrcmp(category->categoryName(), "driver.usb") == 0) {
```

Comment:
Why is this necessary? And why would we only check for `MALIIT_DEBUG` if `driver.usb` is the category? I don't understand this. If this is necessary, please explain it with a comment in the code. Thanks.

Reply:
Duh, used a snippet from the docs and somehow ended up having the actual category name in a git stash. Fixed now.